programs/clearing_house/src/instructions/admin.rs

```diff
@@ -189,7 +198,11 @@ pub fn handle_initialize_spot_market(
         } else {
             MarketStatus::Initialized
         },
-        asset_tier: AssetTier::Collateral,
+        asset_tier: if spot_market_index == QUOTE_SPOT_MARKET_INDEX {
+            AssetTier::Collateral
+        } else {
+            AssetTier::Isolated
```

Comment:
this is so we have to manually make it collateral tier?

Reply:
thought this would be a nice default